src/main/groovy/de/triplet/gradle/play/PlayPublishApkTask.groovy

```diff
@@ -19,10 +19,10 @@ class PlayPublishApkTask extends PlayPublishTask {
     @TaskAction
     publishApks() {
         publish()
-
+        def apkDir = new File(inputFolder.getParentFile().getParentFile(), "apk/${variant.dirName}")
```

Comment:
On a short note without proper review this can be written as:
```
def apkDir = new File(inputFolder.parentFile.parentFile, "apk/${variant.dirName}")
```